### PR TITLE
Migrate individual quicksearch

### PR DESCRIPTION
**Pull request by brmscheiner**

### CODE REVIEW

File: src/components/AuthenticatedAppHeader/IndividualsButton.jsx [hunk 1]
```diff
@@ -168,14 +174,14 @@ export default function IndividualsButton() {
...
-                      href={`/individuals/${resultId}`}
+                      href={`/individuals/${individualGuid}`}
                       noUnderline
                     >
                       {displayString}
                     </Link>
                   }
                   secondary={
-                    <Text variant="caption">{speciesString}</Text>
+                    <Text variant="caption">{`Created on ${createdDate}`}</Text>
```
Comment (Atticus29) [inline src/components/AuthenticatedAppHeader/IndividualsButton.jsx:184]: Why this change?

File: src/components/dataDisplays/DataDisplay.jsx [hunk 2]
```diff
@@ -159,36 +161,45 @@ export default function DataDisplay({
...
+                          <Checkbox
+                            size="small"
+                            checked={visibleColumnNames.includes(
+                              c.name,
+                            )}
+                            onClick={() => {
+                              if (
+                                visibleColumnNames.includes(c.name)
+                              ) {
+                                if (visibleColumnNames.length === 1)
```
Comment (Atticus29) [inline src/components/dataDisplays/DataDisplay.jsx:181]: Why wouldn't we want a table with just one visible column to return anything?

File: src/models/individual/usePostIndividual.js [hunk 3]
```diff
@@ -22,11 +22,7 @@ export default function usePostIndividual() {
         },
       });
       const successful = get(response, 'status') === 200;
-      const newIndividualGuid = get(response, [
-        'data',
-        'result',
-        'id',
-      ]);
+      const newIndividualGuid = get(response, ['data', 'guid']);
```
Comment (Atticus29) [inline src/models/individual/usePostIndividual.js:25]: Just want to confirm that the 'result' is no longer needed here?

**Review by Atticus29 — APPROVED** [review]
A few questions. Looks great!

File: src/models/individual/usePostIndividual.js (hunk 3 above)
Comment (Emily-Ke) [reply src/models/individual/usePostIndividual.js:25]: I think this changed in https://github.com/WildMeOrg/houston/pull/552.

File: src/components/AuthenticatedAppHeader/IndividualsButton.jsx [hunk 4]
```diff
@@ -120,9 +120,9 @@ export default function IndividualsButton() {
             placeholder:
               'Search for an individual by name, alias, or guid',
```
[the remark this answers is not in the document]
Comment (Atticus29) [reply src/components/AuthenticatedAppHeader/IndividualsButton.jsx:121]: Oh yeah. And this is already in en.json as well. And I suppose it might should be changed there as well.

File: src/components/AuthenticatedAppHeader/IndividualsButton.jsx (hunk 1 above)
Comment (brmscheiner) [reply src/components/AuthenticatedAppHeader/IndividualsButton.jsx:184]: Species was no longer there.

File: src/models/individual/usePostIndividual.js (hunk 3 above)
Comment (brmscheiner) [reply src/models/individual/usePostIndividual.js:25]: Correct

File: src/models/individual/useIndividualTermQuery.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import useFetch from '../../hooks/useFetch';
+import { getIndividualTermQueryKey } from '../../constants/queryKeys';
+
+export default function useIndividualTermQuery(searchTerm) {
+  const query = {
+    query_string: {
+      query: `*${searchTerm.toLowerCase()}*`,
```
[the remark this answers is not in the document]
Comment (brmscheiner) [reply src/models/individual/useIndividualTermQuery.js:7]: That's a great question. Not sure if all dialects _can_ be lowercased? I didn't confirm it's necessary for the new ES API, but in Gumby I had to do this or the query string wouldn't match anything if it had capital letters. I'll test and leave this if it's the same, but remove it if possible

File: src/components/dataDisplays/DataDisplay.jsx (hunk 2 above)
Comment (brmscheiner) [reply src/components/dataDisplays/DataDisplay.jsx:181]: So actually the only thing that changed here was the indentation. But this prevents people from hiding the last column of a table when they are showing/hiding columns.

File: src/models/individual/useIndividualTermQuery.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (brmscheiner) [reply src/models/individual/useIndividualTermQuery.js:7]: Ah looks like it's no longer necessary

**Review by Emily-Ke — APPROVED** [review]
